src/components/common/commonCarousel/CommonCarousel.tsx

```diff
@@ -0,0 +1,96 @@
+"use client";
```

Comment:
This is necessary?

Reply:
If I don't use "use client" I get this error
Error: (0 , react__WEBPACK_IMPORTED_MODULE_0__.createContext) is not a function